src/app/components/header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Link = styled.a`
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  font-size: 2.2em;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
```

Comment:
@jp-bbc any chance of some hover and focus CSS, such as text-decoration: underline?

Reply:
There isn't any point doing this here as the component will shortly be changed to have an svg logo rather than text :'( therefore the text styling will be thrown away long before release.